consensus/hotstuff/randombeacon_signer.go

```diff
@@ -0,0 +1,57 @@
+package hotstuff
+
+import "github.com/onflow/flow-go/crypto"
+
+type RandomBeaconSigner interface {
```

Comment:
I am lost again, can you explain why do we have `RandomBeaconSigner` interface which can verify, aggregate and sign and we also have `RandomBeaconAggregator` and `RandomBeaconReconstructor` which are very similar and know only how to aggregate, appreciate use cases as well.